src/messenger_packets.md

```diff
@@ -0,0 +1,147 @@
+Title: Packets of messenger
+
+This is a description of packets of messenger layer of Tox.
+
+## FILE_DATA
+
+FileData packet is used to transfer sender's data file to a friend.
+It holds `file_id` which is one byte long, means that a tox client can send maximum 256 files concurrently.
+Also it means that two friends can send 512 files to each other concurrently.
+
+Serialized form:
+
+Length    | Content
+--------- | ------
+`1`       | `0x52`
+`1`       | `file_id`
+`0..1371` | file data piece
+
+One FileData packet can hold maximum 1371 bytes.
+To send whole file which is bigger than 1371 bytes, messenger's file sending function need to be called many times.
+Multiple calling file sending function of messenger must be done by the user of tox-rs, Tox-rs provides only an api for sending a chunk of file data.
```

Comment:
Change tox-rs to tox protocol everywhere. We describe how proto should work in general not how tox-rs should work.